manim/mobject/mobject.py

```diff
@@ -96,6 +120,37 @@ def __init__(self, color=WHITE, name=None, dim=3, target=None, z_index=0, **kwar
 
         Container.__init__(self, **kwargs)
 
+    @classmethod
+    def get_animation_override(cls, animation_class):
+        if animation_class in cls.animation_overrides:
+            return cls.animation_overrides[animation_class]
+        return None
+
+    @classmethod
+    def _add_intrinsic_animation_overrides(cls):
+        for method_name in dir(cls):
+            if method_name.startswith("__"):  # Preventing attribute errors
+                continue
+            method = getattr(cls, method_name)
+            if hasattr(method, "_override_animation"):
+                animation_class = method._override_animation
+                cls.add_animation_override(animation_class, method)
+
+    @classmethod
+    def add_animation_override(cls, animation_class, override_func):
```

Comment:
Nonblocking, but perhaps a `remove_animation_override` method could be added too.

Reply:
At the moment I have not a lot of time to spare so I would rather not :)